client/my-sites/domains/domain-management/transfer/transfer-to-any-user/transfer-domain-to-any-user.tsx

```diff
@@ -0,0 +1,253 @@
+	selectedDomainName: string;
+	selectedSite: any;
+} ) {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const [ email, setEmail ] = useState( '' );
+	const [ isValidEmail, setIsValidEmail ] = useState( true );
+	const [ errorMessage, setErrorMessage ] = useState( '' );
+
+	const currentRoute = useSelector( getCurrentRoute );
+
+	let selectedDomain: ResponseDomain | undefined;
+	if ( ! isRequestingSiteDomains ) {
+		selectedDomain = getSelectedDomain( { domains, selectedDomainName } );
+	}
+
+	const isLoading = ! hasSiteDomainsLoaded || isRequestingSiteDomains;
+
+	const handleEmailChange = ( event: React.ChangeEvent< HTMLInputElement > ) => {
+		const email = event.target.value;
+
+		setEmail( email );
+
+		if ( email.length > 0 && ! emailValidator.validate( email ) ) {
+			setIsValidEmail( false );
+			setErrorMessage( translate( 'Please enter a valid email address.' ) );
+		}
+
+		setIsValidEmail( true );
+	};
+
+	const handleSubmit = ( event: React.FormEvent< HTMLFormElement > ) => {
+		event.preventDefault();
+
+		if ( email.length > 0 && ! emailValidator.validate( email ) ) {
+			setIsValidEmail( false );
+			setErrorMessage( translate( 'Please enter a valid email address.' ) );
+			return false;
+		}
+
+		wp.req
+			.post( `/sites/${ selectedSite?.ID }/domains/${ selectedDomainName }/transfer-to-any-user`, {
+				email,
+			} )
+			.then(
+				() => {
+					dispatch(
+						successNotice(
+							translate( 'A domain transfer request has been sent to the receiving user.' ),
+							{ duration: 10000, isPersistent: true }
+						)
+					);
+				},
+				() => {
+					dispatch(
+						errorNotice(
+							translate( 'An error occurred while initiating the domain transfer.' ),
+							noticeOptions
+						)
+					);
+				}
+			);
+
+		return false;
+	};
+
+	const hasEmailWithUs =
+		selectedDomain && ( hasTitanMailWithUs( selectedDomain ) || hasGSuiteWithUs( selectedDomain ) );
+
+	const renderForm = () => {
+		if ( ! selectedDomain?.currentUserCanManage ) {
+			return <NonOwnerCard domains={ domains } selectedDomainName={ selectedDomainName } />;
+		}
+
+		if ( selectedDomain?.aftermarketAuction ) {
+			return <AftermarketAutcionNotice domainName={ selectedDomainName } />;
+		}
+
+		if ( selectedDomain?.isRedeemable ) {
+			return <NonTransferrableDomainNotice domainName={ selectedDomainName } />;
+		}
+
+		if ( selectedDomain?.pendingRegistration ) {
+			return (
+				<TransferUnavailableNotice
+					message={ translate(
+						'We are still setting up your domain. You will not be able to transfer it until the registration setup is done.'
+					) }
+				></TransferUnavailableNotice>
+			);
+		}
+
+		return (
+			<Card className="transfer-domain-to-any-user__card">
+				<form onSubmit={ handleSubmit }>
+					<FormFieldset>
+						<FormLabel>{ translate( 'Email of user to transfer to' ) }</FormLabel>
+						<FormTextInput
+							disabled={ isLoading }
+							name="email"
+							noWrap
```

Comment:
I'm seeing this warning in the console
![noWrap](https://github.com/Automattic/wp-calypso/assets/6586048/10d3b52e-47e9-4968-9fea-aad4b789aa35)

Reply:
Not even sure why that's there!

```suggestion
```